HIPs/hip-1.md

```diff
@@ -0,0 +1,271 @@
+
+If you are interested in assuming ownership of an HIP, send a message asking to take over, addressed to both the original author and the HIP editor. If the original author doesn't respond to email in a timely manner, the HIP editor will make a unilateral decision (it's not like such decisions can't be reversed :)).
+
+## HIP Editors
+
+The current HIP editors are
+
+` * Name (@GithubUser)`
+
+
+## HIP Editor Responsibilities
+
+For each new HIP that comes in, an editor does the following:
+
+- Read the HIP to check if it is ready: sound and complete. The ideas must make technical sense, even if they don't seem likely to get to final status.
+- The title should accurately describe the content.
+- Check the HIP for language (spelling, grammar, sentence structure, etc.), markup (GitHub flavored Markdown), code style
+
+If the HIP isn't ready, the editor will send it back to the author for revision, with specific instructions.
+
+Once the HIP is ready for the repository, the HIP editor will:
+
+- Assign an HIP number (generally the PR number or, if preferred by the author, the Issue # if there was discussion in the Issues section of this repository about this HIP)
+
+- Merge the corresponding pull request
+
+- Send a message back to the HIP author with the next step.
+
+Many HIPs are written and maintained by developers with write access to the Handshake codebase. The HIP editors monitor HIP changes, and correct any structure, grammar, spelling, or markup mistakes we see.
+
+The editors don't pass judgment on HIPs. We merely do the administrative & editorial part.
+
+## Style Guide
+
+When referring to an HIP by number, it should be written in the hyphenated form `HIP-X` where `X` is the HIP's assigned number.
+
+## History
+
+This document was derived heavily from [Handshake's HIP-1], which was modified from the BIP process written by Amir Taaki, which in turn was derived from [Python's PEP-0001]. In many places text was simply copied and modified. Although the PEP-0001 text was written by Barry Warsaw, Jeremy Hylton, and David Goodger, they are not responsible for its use in the Handshake Improvement Process, and should not be bothered with technical questions specific to Handshake or the HIP. Please direct all comments to the HIP editors.
+
+August 7, 2020: HIP-1 has been improved and will be placed as a PR.
+
+
+### Bibliography
```

Comment:
Can remove or combine with the preceding section that references EIP BIP and PEP